requirements_dev.txt

```diff
@@ -11,6 +11,7 @@ coverage
 hypothesis
 pytest
 pytest-cov
+pytest-sugar
```

Comment:
I was having a hard time with the default pytest display, this just replaces how the test results get displayed and gives you the exact tests that failed if you want to re-run them (instead of scrolling)

Reply:
There is another pytest plugin (maybe pytest-memory?) which adds an `--lf` command-line option. It will automatically rerun the failing tests. I find it really useful.

Reply:
I think it's called pytest-cache. Will post a link if I find it. I have no objections to your change, just thought you may like this too.

Reply:
I think it may be built into pytest now, rather than being a plug-in.